lib/feature/pokemon_details_page.dart/widgets/evolution_tab/pokemon_evolution_tab.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex_async_redux/api/models/evolution_chain_model.dart';
+import 'package:pokedex_async_redux/api/models/species_model.dart';
+import 'package:pokedex_async_redux/feature/pokemon_details_page.dart/widgets/evolution_tab/widgets/pokemon_evolution_tab_widget.dart';
+
+class EvolutionTab extends StatelessWidget {
+  const EvolutionTab({required this.thisPokeEvo});
+  final EvolutionChain thisPokeEvo;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: EdgeInsets.all(5),
+      child: SingleChildScrollView(
+        child: Center(
+          child: Wrap(
+            children: [
+              PokemonEvolutionTabWidget(thisSpecies: <Species>[thisPokeEvo.evolutionOne!]),
+              if (thisPokeEvo.evolutionTwo.isNotEmpty)
+                Container(
```

Comment:
I assume you're wrapping the Icon with a Container for the spacing. Play around the `alignment` property of the wrap widget. I bet you can find something that will apply even spaces around its children